internal/characters/kazuha/skill.go

```diff
@@ -7,50 +7,60 @@ import (
 	"github.com/genshinsim/gcsim/pkg/core/combat"
 )
 
-var skillPressFrames []int
-var skillHoldFrames []int
+var skillPressFrames [][]int
+var skillHoldFrames [][]int
 
 const (
 	skillPressHitmark = 10
+	skillPressRadius  = 5
 	skillPressCDStart = 8
 	skillHoldHitmark  = 33
+	skillHoldRadius   = 9
 	skillHoldCDStart  = 31
 	particleICDKey    = "kazuha-particle-icd"
 )
```

Comment:
super nit: would be best if we reduced `skillPress` and `skillHold` to a single function since the same logic is being duplicated between them (just with different values).

Naive thought on this would be you need to pass through:
* `radius`
* stats skill multiplier array
* `hitmark`
* `particles`
* `cd` (change C1 to subtract 10%)
* `cdStartDelay`
* frames (for shortest action, either separate variable or do loop to find shortest)
* `glide`

Since all of the information (except `glide`) is completely static, rather than having many `skillHold` and `skillPress` variables, can define something like a `SkillInfo` struct that you create a `hold` and `press` instance within `init` (has to be init because golang very cool and doesnt allow constant structs).

Result would be your generic function with a header like:
```go
func (c *char) skillExecute(info *SkillInfo, glide int) action.ActionInfo {
  // ...
}
```

Reply:
Out of scope of the PR. If I was to do this then I would also take a look at all of the other skillTap/skillHold implementations.